code/datums/wounds/cuts.dm

```diff
@@ -0,0 +1,259 @@
+	var/datum/scar/highest_scar
+
+/datum/wound/brute/cut/apply_wound(obj/item/bodypart/L, silent = FALSE, datum/wound/brute/cut/old_wound = null)
+	. = ..()
+	blood_flow = initial_flow
+	if(old_wound)
+		blood_flow = max(old_wound.blood_flow, initial_flow)
+		if(old_wound.severity > severity && old_wound.highest_scar)
+			highest_scar = old_wound.highest_scar
+			old_wound.highest_scar = null
+		if(old_wound.current_bandage)
+			current_bandage = old_wound.current_bandage
+			old_wound.current_bandage = null
+
+	if(!highest_scar)
+		highest_scar = new
+		highest_scar.generate(L, src, add_to_scars=FALSE)
+
+/datum/wound/brute/cut/remove_wound(ignore_limb, replaced)
+	if(!replaced && highest_scar)
+		already_scarred = TRUE
+		highest_scar.lazy_attach(limb)
+	return ..()
+
+/datum/wound/brute/cut/get_examine_description(mob/user)
+	if(!current_bandage)
+		return ..()
+
+	var/bandage_condition = ""
+	// how much life we have left in these bandages
+	switch(current_bandage.absorption_capacity)
+		if(0 to 1.25)
+			bandage_condition = "nearly ruined "
+		if(1.25 to 2.75)
+			bandage_condition = "badly worn "
+		if(2.75 to 4)
+			bandage_condition = "slightly bloodied "
+		if(4 to INFINITY)
+			bandage_condition = "clean "
+	return "<B>The cuts on [victim.p_their()] [limb.name] are wrapped with [bandage_condition] [current_bandage.name]!</B>"
+
+/datum/wound/brute/cut/receive_damage(wounding_type, wounding_dmg, wound_bonus)
+	if(wounding_type == WOUND_SHARP)
+		blood_flow += 0.05 * wounding_dmg
+
+/datum/wound/brute/cut/handle_process()
+	if(victim.reagents && victim.reagents.has_reagent(/datum/reagent/medicine/coagulent))
+		blood_flow -= 0.25
+	if(current_bandage)
+		if(clot_rate > 0)
+			blood_flow -= clot_rate
+		blood_flow -= current_bandage.absorption_rate
+		current_bandage.absorption_capacity -= current_bandage.absorption_rate
+		if(current_bandage.absorption_capacity < 0)
+			victim.visible_message("<span class='danger'>Blood soaks through \the [current_bandage] on [victim]'s [limb.name].</span>", "<span class='warning'>Blood soaks through \the [current_bandage] on your [limb.name].</span>", vision_distance=COMBAT_MESSAGE_RANGE)
+			QDEL_NULL(current_bandage)
+			treat_priority = TRUE
+	else
+		blood_flow -= clot_rate
+
+	if(blood_flow > highest_flow)
+		highest_flow = blood_flow
+
+	if(blood_flow < minimum_flow)
+		if(demotes_to)
+			replace_wound(demotes_to)
+		else
+			to_chat(victim, "<span class='green'>The cut on your [limb.name] has stopped bleeding!</span>")
+			qdel(src)
+
+/* BEWARE, THE BELOW NONSENSE IS MADNESS. bones.dm looks more like what I have in mind and is sufficiently clean, don't pay attention to this messiness */
+
+/datum/wound/brute/cut/treat(obj/item/I, mob/user)
+	if(istype(I, /obj/item/gun/energy/laser))
+		las_cauterize(I, user)
+	else if(I.tool_behaviour == TOOL_CAUTERY || I.get_temperature() > 300)
```

Comment:
er, no items that are on the list(s) of valid items for treating cuts would have a temperature value (above 300), no?

Reply:
It allows any item with a temperature about 300 to act as a cautery